src/app/screens/connectors/ChooseConnectorPath/index.tsx

```diff
@@ -1,66 +1,135 @@
 
   return (
-    <div className="relative mt-10 lg:grid lg:gap-8 text-center">
+    <div className="relative mt-10 lg:grid lg:gap-8">
       <div className="relative">
-        <div className="grid lg:grid-cols-2 gap-10 mb-4">
+        <h1 className="text-2xl font-bold dark:text-white text-center mb-8">
+          {t("title")}
+        </h1>
+        <div className="grid lg:grid-cols-3 gap-5 mb-4">
           <ConnectorPath
-            title={t("alby.title")}
-            description={t("alby.description")}
+            title={t("other.title")}
+            icon={
+              <div className="grid grid-cols-2 gap-1">
+                <img
+                  src="assets/icons/lnd.png"
+                  className="w-[18px] h-[18px] rounded-md"
+                />
+                <img
+                  src="assets/icons/umbrel.png"
+                  className="w-[18px] h-[18px] rounded-md"
+                />
+                <img
+                  src="assets/icons/btcpay.svg"
+                  className="w-[18px] h-[18px] rounded-md"
+                />
+                <img
+                  src="assets/icons/core_ln.png"
+                  className="w-[18px] h-[18px] rounded-md"
+                />
+              </div>
+            }
+            description={t("other.description")}
             content={
-              <img src={alby} alt="logo" className="inline rounded-3xl w-32" />
+              <>
+                <FeatureItem type="success">{t("other.point1")}</FeatureItem>
+                <FeatureItem type="success">{t("other.point2")}</FeatureItem>
+                <FeatureItem type="success">{t("other.point3")}</FeatureItem>
+                <FeatureItem type="disabled">{t("other.point4")}</FeatureItem>
+              </>
+            }
+            actions={
+              <Link to="choose-connector" className="flex flex-1">
+                <Button tabIndex={-1} label={t("other.connect")} primary flex />
+              </Link>
             }
-            actions={<ConnectAlby />}
           />
           <ConnectorPath
-            title={t("other.title")}
-            description={t("other.description")}
+            title={t("voltage.title")}
+            icon={<img src="assets/icons/voltage.png" className="w-10 h-10" />}
+            description={t("voltage.description")}
             content={
+              <>
+                <FeatureItem type="success">{t("voltage.point1")}</FeatureItem>
+                <FeatureItem type="success">{t("voltage.point2")}</FeatureItem>
+                <FeatureItem type="success">{t("voltage.point3")}</FeatureItem>
+                <FeatureItem type="success">{t("voltage.point4")}</FeatureItem>
+              </>
+            }
+            actions={
               <Link
-                to="choose-connector"
-                className="flex flex-wrap gap-6 w-72 lg:w-80 mx-auto my-5 items-center"
-                tabIndex={-1}
+                to="https://info.getalby.com/voltage"
```

Comment:
is this a URL that we easily can redirect? otherwise I'd recommend getalby.com/voltage

Reply:
What do you think about copying the LND connector screen and using that with custom Voltage code? 

we can add a link to the guide there and this could be the image:
![CleanShot 2023-12-22 at 00 16 10@2x](https://github.com/getAlby/lightning-browser-extension/assets/318/50cd376d-be64-4ae3-85e0-9e8af808b9d5)